packages/api-client/src/store/import-spec.ts

```diff
@@ -10,7 +10,7 @@ import { toRaw } from 'vue'
 /** Maps the specs by URL */
 export const specDictionary: Record<string, { hash: number; schema: OpenAPIV3.Document | OpenAPIV3_1.Document }> = {}
 
-type ImportSpecFileArgs = ImportSpecToWorkspaceArgs & Pick<ApiReferenceConfiguration, 'servers'>
+type ImportSpecFileArgs = ImportSpecToWorkspaceArgs & Pick<ApiReferenceConfiguration, 'servers' | 'proxyUrl'>
```

Comment:
we weren't using the proxy before at all? Oh my. I guess from references we passed down the spec

Reply:
wait the proxy is working from before

Reply:
the proxy is used for requests, but wasn’t used to fetch external references, that’s all 👌